zebra-chain/src/primitives/zcash_primitives.rs

```diff
@@ -350,7 +369,7 @@ impl From<zcash_primitives::consensus::Network> for Network {
     fn from(network: zcash_primitives::consensus::Network) -> Self {
         match network {
             zcash_primitives::consensus::Network::MainNetwork => Network::Mainnet,
-            zcash_primitives::consensus::Network::TestNetwork => Network::Testnet,
+            zcash_primitives::consensus::Network::TestNetwork => Network::new_default_testnet(),
```

Comment:
This _may_ cause problems, because `TestNetwork` could also mean Regtest. Double-check where it's used, and consider replacing this conversion with a regular method.